mountpoint-s3/src/prefix.rs

```diff
@@ -0,0 +1,65 @@
+use std::fmt::Display;
+use std::str::FromStr;
+
+use anyhow::anyhow;
+
+/// A prefix string ending in `/`, or the empty string
+#[derive(Debug, Clone, Default)]
+pub struct Prefix {
+    path: String,
+}
```

Comment:
I think you're right that we should just treat `None` and `Prefix("")` the same way, so we should get rid of `Option<Prefix>` everywhere. That also simplifies a few other things.

Reply:
Removed last usage of Option<Prefix> in CliArgs (using default_value = "" instead)